pulp_rpm/app/management/commands/rpm-datarepair.py

```diff
@@ -22,6 +30,8 @@ def handle(self, *args, **options):
 
         if issue == "2460":
             self.repair_2460()
+        elif issue == "2735":
+            self.repair_2735()
```

Comment:
It sounds like we're just making up data in some places. I'm not sure that's a great idea. We should just extend the new stage (which did end up being necessary) to cover this, instead - if we have verified that it can indeed happen in normal operation that is.

https://github.com/pulp/pulp_rpm/pull/2784/files#diff-fb406f31027edc1c5b65abe7635bbd2830a7cc05c24a487344d7649017d14628R1594

Reply:
@dralley Are you suggesting we move the fixups here, from datarepair into RpmQueryExisting? I *think* I agree.

Reply:
@dralley @ggainey , closing this PR as already working on extending sync stage.